e2e/components/Axe.test.ts

```diff
@@ -0,0 +1,49 @@
+import {test, expect} from '@playwright/test'
+// @ts-ignore since this file is generated in the ci to generate this file run npx build storybook in packages/react
+import componentsConfig from '../../packages/react/storybook-static/index.json'
+import {visit} from '../test-helpers/storybook'
+import {themes} from '../test-helpers/themes'
+
+/**
+ * These stories should not be tested in the CI because they are stress-tests and
+ * perform slowly
+ */
+const SKIPPED_TESTS = [
+  'components-treeview-features--stress-test',
+  'components-treeview-features--contain-intrinsic-size',
+]
+
+type Component = {
+  name: string
+}
+
+const {entries} = componentsConfig
+
+test.describe('@aat', () => {
+  for (const [id, entry] of Object.entries(entries as Record<string, Component>)) {
+    const {name} = entry
+    // remove parentheses and slashes from the name to avoid playwright file issues
+    // eslint-disable-next-line no-useless-escape
+    const cleanedName = name.replaceAll(/[\(\)\/]/g, '')
+
+    test.describe(id, () => {
+      for (const theme of themes) {
+        test.describe(theme, () => {
+          test(cleanedName, async ({page}) => {
+            if (SKIPPED_TESTS.includes(id)) {
+              return
+            } else {
+              await visit(page, {
+                id,
+                globals: {
+                  colorScheme: themes[0],
+                },
+              })
+              await expect(page).toHaveNoViolations()
+            }
+          })
+        })
+      }
+    })
+  }
```

Comment:
Just a couple of suggestions, let me know what you think!

- Skip tests at the outermost loop level instead of having empty tests
- Update `colorScheme` to point  to `theme`
- Add `@aat` tag at the test-level

```suggestion
  for (const [id, entry] of Object.entries(entries as Record<string, Component>)) {
    if (SKIPPED_TESTS.includes(id)) {
      continue;
    }

    const {name} = entry
    // remove parentheses and slashes from the name to avoid playwright file issues
    // eslint-disable-next-line no-useless-escape
    const cleanedName = name.replaceAll(/[\(\)\/]/g, '')

    test.describe(id, () => {
      for (const theme of themes) {
        test.describe(theme, () => {
          test(`${cleanedName} @aat`, async ({page}) => {
            await visit(page, {
              id,
              globals: {
                colorScheme: theme,
              },
            })
            await expect(page).toHaveNoViolations()
          })
        })
      }
    })
  }
```